parquet/src/arrow/arrow_reader/selection.rs

```diff
@@ -116,6 +118,62 @@ impl RowSelection {
         Self { selectors }
     }
 
+    /// Given an offset index, return a mask indicating which pages are selected along with their locations by `self`
+    pub fn page_mask(
```

Comment:
```suggestion
    pub(crate) fn page_mask(
```

I don't think this likely to be useful outside the crate